scripts/mkimg.base.sh

```diff
@@ -256,6 +256,10 @@ create_image_iso() {
 		grub-mkrescue --output ${ISO} ${DESTDIR} -follow-links \
 			-sysid LINUX \
 			-volid "alpine-$PROFILE $RELEASE $ARCH"
+	elif [ "$ARCH" = "s390x" ]; then
+		for _image in vmlinuz initramfs modloop; do
+			cp ${DESTDIR}/boot/$_image-$kernel_flavors ${OUTDIR}
+		done
```

Comment:
Oh, I see what you do here. You don't create iso image, only kernel. The idea is good but I think we should solve it differently, because:
- the kernel, initramfs and modloop are without version number, which means they will be overwritten next version.
- this is something we may want for netboot of the other arches as well

I did the 3.8.0 rc1 without this and it generated iso image for s390x. you can use `uniso` (from `alpine-conf` package) or `7z` to extract the files from the iso.

I also found out that suse has a perl script to make iso image bootable with zipl: https://github.com/openSUSE/mksusecd/blob/master/isozipl

Reply:
Agreed. As we talked in IRC, I comment here again for completeness. The ISO might be useful for LPAR boot but we don't have the hardware atm, and we are targeting KVM and z/VM so kernel,initramfs, modloop are enough.
We can still do ISO for KVM thought if I'm not wrong but I don't have time to do it for now, might be 3.8.1.
I'm going to close this pr. Many thanks.